doc/classes/EditorPlugin.xml

```diff
@@ -587,6 +587,24 @@
 				Returns the [PopupMenu] under [b]Scene &gt; Export As...[/b].
 			</description>
 		</method>
+		<method name="get_node_3d_rotate_snap">
+			<return type="float" />
+			<description>
+				Returns the amount of degrees the 3D editor's rotational snapping is set to.
+			</description>
+		</method>
+		<method name="get_node_3d_scale_snap">
+			<return type="float" />
+			<description>
+				Returns the amount of units the 3D editor's scale snapping is set to.
+			</description>
+		</method>
+		<method name="get_node_3d_translate_snap">
+			<return type="float" />
+			<description>
+				Returns the amount of units the 3D editor's translation snapping is set to.
+			</description>
+		</method>
```

Comment:
Wouldn't this make more sense to expose in `EditorInterface` than `EditorPlugin`?

Reply:
Yes, I agree this makes more sense in EditorInterface.